app/src/addr.h

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
-*   (c) 2018 Zondax GmbH
+*   (c) 2018 - 2022 Zondax AG
```

Comment:
Zondax changed its company legal form?

Reply:
It seems indeed. I copied this one from another zondax app...